tests/searchcommands/test_configuration_settings.py

```diff
@@ -81,7 +81,7 @@ def generate(self):
 
```

Comment:
I had to change this to

 [('generating', True), ('type', 'stateful')])

to get the test to pass.